cli/Cargo.toml

```diff
@@ -64,8 +64,8 @@ serde = { version = "1.0.116", features = ["derive"] }
 sys-info = "0.7.0"
 sourcemap = "6.0.1"
 swc_bundler = "=0.14.1"
```

Comment:
Can we upgrade the bundler? It requires https://github.com/swc-project/swc/blob/8e07d7c7a6366801909565bf7e2b8d5762dd3edd/bundler/tests/deno.rs#L254-L289.

Reply:
Yes, I will do that, but it requires some changes need to look into that

Reply:
Yes I upgraded the `import.meta` hook API in https://github.com/swc-project/swc/pull/1195, I linked the new required implementation above.

Reply:
Okay, fixed, PTAL